README.md

```diff
@@ -25,117 +25,120 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
+See [below](#how-to-add-new-modules) if you are interested in writing a new module and implementing your own target device.
 
 # Usage
 
 ## Installation
 
 ~~Install I2Cwrapper from the Arduino library manager (not available yet).~~ Until I2Cwrapper is picked up by the Arduino library manager, you'll have to [install it manually](https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries#manual-installation) by copying the I2Cwrapper folder to your Arduino library folder and (re)starting the Arduino software.  You'll find an I2Cwrapper examples folder after successful installation.
 
-## Configuring and uploading the firmware
+## Target Side: Configuring and Uploading the Firmware
 
 * **Open [firmware.ino](https://github.com/ftjuh/AccelStepperI2C/blob/controller/firmware/firmware.ino)**  from the examples menu of the Arduino editor, you'll find it in the the I2Cwrapper submenu. It will open multiple tabs, among them one for each available module in the firmware subfolder.
-* Go to  the `firmware_modules.h` tab and **select the modules** you want by (un)commenting them. For a first test, start with the PinI2C module, it is the simplest and doesn't need any extra hardware.
+* Go to  the [`firmware_modules.h`](firmware/firmware_modules.h) tab and **select the modules** you want by (un)commenting them. For a first test, start with the `PinI2C` module, it is the simplest and doesn't need any extra hardware.
 * You can **save a local copy** of the firmware. Don't forget, though, that your local copy won't be updated in future releases which might result in conflicts after a library upgrade.
 * **Compile and upload** to your target device.
 
-## Testing the firmware
+### Testing the Firmware
 
 The target device is now ready. To test it, you can use one of the example sketches:
 
 * (optional, only tested for Linux) Open a completely **new instance** of the Arduino environment from your start menu. That way, you can connect target and controller devices at the same time without the need to change USB ports for uploading and serial output.
-* Depending on the module(s) you selected, load **one of the examples** from the example folder and upload it to your controller device. Use `Pin_control.ino` for a first test with the PinI2C module.
+* Depending on the module(s) you selected, load **one of the examples** from the example folder and upload it to your controller device. Use `Pin_control.ino` for a first test with the `PinI2C` module.
 * **Configure and upload** the example sketch or your own controller sketch to another Arduino-like (controller).
-* **Connect the I2C bus** of both devices. Don't forget  I2C pullups and, if needed, level-shifters. Also connect +V <-> +V to power one board from the other, if needed.
+* **Connect the I2C bus** of both devices. Don't forget  I2C pullups and, and if needed, level-shifters. Also connect GND <-> GND to power one board from the other, if needed.
```

Comment:
`GND` right?

Reply:
No, I meant V, in the case that only one of the devices is connected to a power source. I assumed that connecting GND is part of "Connect the I2C bus", as it won't work only with SDA and SCL. Should we make this more explicit?